lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -6031,6 +6056,80 @@ CompilerType TypeSystemClang::GetVirtualBaseClassAtIndex(
   return CompilerType();
 }
 
+std::optional<int64_t>
+TypeSystemClang::TryToGetBaseOffset(const clang::CXXRecordDecl *derived,
```

Comment:
If you don't expect `derived` to be `nullptr`, make it a reference instead.
It seems that `base` could be `nullptr`, but your code seems to ignore it (or maybe intentionally let is pass through?)